apps/dashboard/src/app/(app)/team/page.tsx

```diff
@@ -1,10 +1,7 @@
-import { getLastVisitedTeamOrDefaultTeam } from "@/api/team";
-import { notFound, redirect } from "next/navigation";
+import { getLastVisitedTeam } from "@/api/team";
+import { redirect } from "next/navigation";
 
 export default async function TeamRootPage() {
-  const team = await getLastVisitedTeamOrDefaultTeam();
-  if (!team) {
```

Comment:
Previously this would redirect to the notFound page if the token was invalid. Now it redirects to /team/~ to handle it there.
Basically I think `/team` should always redirect to another team page, and let those pages handle the logged out or invalid auth cases.